ex01.exs

```diff
@@ -74,14 +74,23 @@ end
 defmodule Ex01 do
 
   def counter(value \\ 0) do
-    # ...your code
+    receive do
+      { :next, from} ->
+        send from, {:next_is, value}
+        counter(value + 1)
+        
+    end
   end
 
   def new_counter(initial_value \\ 0) do
-    # ... your code
+    count = spawn(Ex01, :counter, [initial_value])
```

Comment:
~~~
warning: variable "count" is unused
~~~

I treat warnings as errors, and fix them in the same way, because my experience is that they often are early signs of a bug.

In this case, the warning is benign, but the principle holds.

# -1